File: ProgramasPython/secao06/Exercicio08.py
numero = int(input("Insira um nº inteiro: "));

if numero == 0:
    
    print("O nº digitado é neutro.");
    
elif numero % 2 == 0:
    
    if numero > 0:
        
        print("O nº {0} é par e positivo.".format(numero))
        
    else:
        
        print("O nº {0} é par e negativo.".format(numero))
        
else:
    
    if numero > 0:
        
        print("O nº {0} é ímpar e positivo.".format(numero))
        
    else:
        
        print("O nº {0} é ímpar e negativo.".format(numero))
    

File: ProgramasJava/src/br/com/geekuniversity/secao06/Exercicio08.java
package br.com.geekuniversity.secao06;

import java.util.*;

public class Exercicio08 {

	public static void main(String[] args) {
		
		int n;
		Scanner tela = new Scanner(System.in);
		
		System.out.println("Digite um número qualquer:");
		n = tela.nextInt();
		
		if(n == 0) {

			System.out.printf("%nO nº %d é neutro.", n);
			
		}
		
		else if (n % 2 == 0) {
			
			if(n > 0) {
			
				System.out.printf("%nO nº %d é par e positivo", n);
				
			}
			
			
			else {
				
				System.out.printf("%nO nº %d é par e negativo", n);
				
			}
			
		}
		
		else if (n % 2 != 0) {
			
			if(n > 0) {
			
				System.out.printf("%nO nº %d é ímpar e positivo", n);
				
			}
			
			
			else {
				
				System.out.printf("%nO nº %d é ímpar e negativo", n);
				
			}
			
		}
		
		tela.close();
		
	}

}
